hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServer.java

```diff
@@ -77,10 +78,9 @@ protected void configureServlets() {
       LOG.info("Starting Recon server");
       httpServer.start();
 
-      //Start Ozone Manager Service that pulls data from OM.
-      OzoneManagerServiceProvider ozoneManagerServiceProvider = injector
-          .getInstance(OzoneManagerServiceProvider.class);
-      ozoneManagerServiceProvider.start();
+      getOzoneManagerServiceProvider().start();
```

Comment:
Service providers generally should not have a life cycle method. Let's start a service provided by the OzoneManagerServiceProvider, or refactor to an inner class of it.

Reply:
The class should be renamed to something like "ReconOzoneManager" since its role is more akin to a manager than a service provider. I can do that in another patch since it will touch all files.